benchmarks/jmh/pom.xml

```diff
@@ -6,16 +6,17 @@
         <artifactId>root</artifactId>
         <groupId>com.yahoo.oak</groupId>
         <version>0.1.4-SNAPSHOT</version>
```

Comment:
I've modified the root pom to have `<dependencyManagement>` element so that submodules can include oak core dependency without explicit version.
Using properties for versioning is problematic. Instead, we can use versions-maven-plugin:
` mvn versions:set -DnewVersion=0.1.5-SNAPSHOT`